Missing definition of RAILCOM_CUTOUT_POST_DELTA_USEC

There appears to be a missing definition for the constant RAILCOM_CUTOUT_POST_DELTA_USEC.

Other similarly named constants are defined in  boards/ti-ek-tm4c123gxl-launchpad/HwInit.cxx, so is this an oversight?

Output from the build process follows below (the lm4f120xl uses the tm4c123glx HWInit.cxx file):

make -C freertos.armv7m.ek-lm4f120xl all
make[4]: Entering directory '/home/sid/git/openmrn/applications/async_blink/targets/freertos.armv7m.ek-lm4f120xl'
/opt/armgcc/default/bin/arm-none-eabi-g++ -c -Os -fno-strict-aliasing -fno-strength-reduce -fomit-frame-pointer -fdata-sections -ffunction-sections -g -MD -MP -mcpu=cortex-m3 -mthumb -mfloat-abi=soft -Wall -Werror -Wno-unknown-pragmas -fdata-sections -ffunction-sections -fno-builtin -fno-stack-protector -mfix-cortex-m3-ldrd -D__FreeRTOS__ -DGCC_ARMCM3 -specs=nano.specs -std=c++14 -D_ISOC99_SOURCE -D__STDC_FORMAT_MACROS -fno-exceptions -fno-rtti -Wsuggest-override -Wno-psabi  -DPART_TM4C1233H6PM -DSNIFF_ON_SERIAL  -I/opt/TivaWare/default -I/home/sid/git/openmrn/src/freertos_drivers/ti -I/opt/FreeRTOS/default/FreeRTOS/Source/include -I/opt/FreeRTOS/default/FreeRTOS/Source/portable/GCC/ARM_CM3 -I/home/sid/git/openmrn/include/freertos -idirafter /home/sid/git/openmrn/include/freertos_select -I/home/sid/git/openmrn/src/freertos_drivers/common -I. -I/home/sid/git/openmrn/src/ -I/home/sid/git/openmrn/include -I/home/sid/git/openmrn/applications/async_blink -DBOARD_LAUNCHPAD_EK -MD -MF HwInit.d /home/sid/git/openmrn/applications/async_blink/targets/freertos.armv7m.ek-lm4f120xl/HwInit.cxx -o HwInit.o
In file included from /home/sid/git/openmrn/applications/async_blink/targets/freertos.armv7m.ek-lm4f120xl/HwInit.cxx:56:
/home/sid/git/openmrn/src/freertos_drivers/ti/TivaDCC.hxx: In instantiation of 'TivaDCC<HW>::TivaDCC(const char*, RailcomDriver*) [with HW = DccHwDefs]':
/home/sid/git/openmrn/applications/async_blink/targets/freertos.armv7m.ek-lm4f120xl/HwInit.cxx:313:67:   required from here
/home/sid/git/openmrn/src/freertos_drivers/ti/TivaDCC.hxx:1088:36: error: 'RAILCOM_CUTOUT_POST_DELTA_USEC' is not a member of 'DccHwDefs'
     static_assert((5 * 56 * 2 - 56 + HW::RAILCOM_CUTOUT_POST_DELTA_USEC) >
                   ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/sid/git/openmrn/src/freertos_drivers/ti/TivaDCC.hxx:1091:30: error: 'RAILCOM_CUTOUT_POST_DELTA_USEC' is not a member of 'DccHwDefs'
     target = 5 * 56 * 2 - 56 + HW::RAILCOM_CUTOUT_POST_DELTA_USEC;
              ~~~~~~~~~~~~~~~~^~~~
/home/sid/git/openmrn/etc/prog.mk:270: recipe for target 'HwInit.o' failed
make[4]: *** [HwInit.o] Error 1

--- a/boards/st-stm32f103rb-cmsis/Stm32Can.cxx
+++ b/boards/st-stm32f103rb-cmsis/Stm32Can.cxx
@@ -49,6 +49,12 @@
 extern "C" {
 void USB_HP_CAN1_TX_IRQHandler(void);
 void USB_LP_CAN1_RX0_IRQHandler(void);
+
+static inline void SetInterruptPriority(uint32_t irq, uint8_t priority)
+{
+    NVIC_SetPriority((IRQn_Type)irq, priority >> (8U - __NVIC_PRIO_BITS));
+}
+
 }
 
 class Stm32CanDriver : public Can

--- a/boards/ti-ek-tm4c123gxl-launchpad/HwInit.cxx
+++ b/boards/ti-ek-tm4c123gxl-launchpad/HwInit.cxx
@@ -202,6 +202,9 @@ const uint32_t RailcomDefs::UART_PERIPH[] = {SYSCTL_PERIPH_UART1};
 
 static TivaRailcomDriver<RailcomDefs> railcom_driver("/dev/railcom");
 
+// If 1, enabled spread spectrum randomization of the DCC timings.
+uint8_t spreadSpectrum = 0;
+
 struct DccHwDefs {
   /// base address of a capture compare pwm timer pair
   static const unsigned long CCP_BASE = TIMER0_BASE;
@@ -218,6 +221,7 @@ struct DccHwDefs {
   static const int RAILCOM_CUTOUT_START_DELTA_USEC = -20;
   static const int RAILCOM_CUTOUT_MID_DELTA_USEC = 0;
   static const int RAILCOM_CUTOUT_END_DELTA_USEC = -10;
+  static const int RAILCOM_CUTOUT_POST_DELTA_USEC = -10;
 
   /** These timer blocks will be synchronized once per packet, when the
    *  deadband delay is set up. */

